### ISSUE TITLE
Question about intended goal/behavior.

### ISSUE THREAD

**Issue by PDCWolf** [Beyhum/fake-or-fact#1]
As it is designed right now, this website evaluates your capacity to stay informed, instead of evaluating the capacity to detect fake news. My point is that headline paraphrasing is not enough to detect if an article is fake or not, it'll only appeal to the user's memory if they've seen it, or blind guessing if they haven't, as there's 0 context.

**Comment by Beyhum** [comment]
Yes that's a very valid point, but unfortunately I'm not sure I can use an article's entire body without asking for permission for every one of them.
But either way the site's more about trying to show how surreal/absurd news has become rather than a test of skill!
